pyds8k/resources/ds8k/v1/flashcopy.py

```diff
@@ -43,13 +43,10 @@ def _add_details(self, info, force=False):
         super(FlashCopy, self)._add_details(info, force=force)
 
         # Temporarily, remove this line when flashcopy resource has id field.
-        self._id = self.representation['id'] = '{}:{}'.format(
-            info['sourcevolume']['id'],
-            info['targetvolume']['id']
-        )
+        self._id = self.representation['id']
```

Comment:
if this change is about the [above comment in code](https://github.com/IBM/pyds8k/blob/21f7f14edc2e0a0251f130b646351057793b5fa6/pyds8k/resources/ds8k/v1/flashcopy.py#L45), then please remove said comment.
if not, then why do we want this change?

Reply:
done